src/workflow/elements/eventListeners/EventListener.js

```diff
@@ -0,0 +1,21 @@
+const Joi = require('joi')
+const Element = require('../Element')
+
+class EventListener extends Element {
+
+  static get SCHEMA() {
+    return super.SCHEMA.concat(Joi.object({
+      interrupting: Joi.boolean().default(true),
+      phaseBound: Joi.boolean().default(false),
```

Comment:
Would there be any downsides to treating anything with a phaseId of null as unbound and everything with a phaseId as bound ?

Reply:
Hmm, it would be less explicit, but I like the idea of having only one key to update; I will add your idea.